src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -1131,6 +1140,16 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         });
     };
 
+    hideRepliesForFeedItem = (feedItemId: string) => {
+        const { feedItems } = this.state;
+        const feedItem = feedItems.find((item: FeedItem) => item.id === feedItemId);
+        const { replies } = feedItem;
+        if (replies.length > 1) {
+            const lastReply = feedItem.replies.slice(-1);
```

Comment:
`replies` has already been destructured. Consider using it directly instead of accessing via sub property

`const lastReply = replies.slice(-1);`